data/campaigns/Winds_of_Fate/scenarios/11_Confrontation.cfg

```diff
@@ -0,0 +1,674 @@
+    [event]
+        name=turn 10 end
+        [fire_event]
+            name=burner
+        [/fire_event]
+    [/event]
+
+    [event]
+        name=die
+        [filter]
+            id=Karron
+        [/filter]
+        [message]
+            speaker=Gorlack
+            message= _ "Karron.
+
+My Windpath spared one friend... only to cut down another..."
+        [/message]
+        [endlevel]
+            result=defeat
+        [/endlevel]
+    [/event]
+
+    # If a unit opposed to Karron moves beside him, see if Karron is now surrounded by his opponents. If he is, then they capture him.
+    [event]
+        name=moveto
+        first_time_only=no
+        [filter]
+            side=1
+            [filter_adjacent]
+                id=Karron
+                [filter_adjacent]
+                    count=6
+                    is_enemy=yes
+                [/filter_adjacent]
+            [/filter_adjacent]
+        [/filter]
+        [message]
+            speaker=Karron
+            message= _ "You have one last victory over me...  Finish it."
+        [/message]
+        [message]
+            speaker=Gorlack
+            image_pos=right
+            mirror=yes
+            message= _ "Nay."
+        [/message]
+        [message]
+            speaker=Karron
+            message= _ "......"
+        [/message]
+        [message]
+            speaker=Gorlack
+            image_pos=right
+            mirror=yes
+            message= _ "Karron, I banish you from this greatland, from these far lands of the east.
+Not as the victor vanquishing a rival as it was in the contention.
+
+As a friend seeking to stay you from the turmoil that becomes..."
+        [/message]
+        [message]
+            speaker=Gorlack
+            image_pos=right
+            mirror=yes
+            message= _ "I strove to prevent a war between the flights.
+Yet traded it for a war of an unknown kind.
+
+The hungry flights arriving in numbers will call a hunt far greater than even ours.
+Yet it will not be a hunt.
+
+It will be a war with prey."
+        [/message]
+        [message]
+            speaker=Karron
+            message= _ "Prey can no more make war on us than could the clouds."
+        [/message]
+        [message]
+            speaker=Gorlack
+            image_pos=right
+            mirror=yes
+            message= _ "They already have, as we all bore witness.
+
+Perhaps half the coming flights will be scattered deep into the greatland.
+Where they will meet fates unknown as it was with the Flight of Vladnir.
+Only through defeat may some learn these creatures are much more than prey.
+
+Verily, these creatures are fellow Contenders for dominance over the world itself.
+With this greatland as our arena.
+They are our equals until we <b>prove</b> ourselves their betters.
+
+We shall need allies from among them in this great Contention.
+Like our gentle saurians.
+So too the deep dwellers with whom we have now a pax.
```

Comment:
Unless you were going for 'pax' as in Latin for peace, I think this should be 'pact'.

Reply:
Even if they were going for Latin, it sounds weird. I'd change it to "pact" regardless.

Reply:
Alright, will change it to pact.